assets/css/v2/bus_eink/evergreen/image.scss

```
@@ -0,0 +1,65 @@
+.evergreen-content-image__container {
+  overflow: hidden;
+}
+
+.screen-normal__medium-flex {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
```

Looks like you just need to make this change, based on Mindy's feedback:
```suggestion
    border-radius: 32px;
```